shared/constants/app.ts

```diff
@@ -69,6 +69,9 @@ export const SNAP_DIALOG_TYPES = {
   [DialogType.Confirmation]: MESSAGE_TYPE.SNAP_DIALOG_CONFIRMATION,
   [DialogType.Prompt]: MESSAGE_TYPE.SNAP_DIALOG_PROMPT,
 };
+
+export const SNAP_REGISTRY_URL =
```

Comment:
This should be called something else, otherwise no-one will be able to tell the difference between the Snap Registry and the Keyring Snap Registry